draft-ietf-tls-esni.md

```diff
@@ -930,6 +959,28 @@ in-band when possible, such as through the use of OCSP stapling, and clients
 SHOULD take steps to minimize or protect such requests during certificate
 validation.
 
+## Abuse of ECH Acceptance Signal
+
+To signal acceptance, the backend server overwrites 8 bytes of its
+ServerHello.random with a value derived from the ClientHelloInner.random. (See
+{{backend-server-behavior}} for details.) In the presence of an active
+adversary, this behavior increases the likelihood of the ServerHello.random
```

Comment:
I understand that technically the active adversary can try to exploit this to find matching sessions, but I think the notion might just confuse readers.

```suggestion
{{backend-server-behavior}} for details.) This behavior increases the likelihood of the ServerHello.random
```

Reply:
Agreed. Done.